src/index.ts

```diff
@@ -14,12 +14,41 @@ const app = express();
 app.use(expressSanitizer());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+const getSecretForTag = function(req, payload, done){
+  const { tag } = req.query;
+
+  // fallback to the default JWT_SECURITY_SALT if the new variable isn't set up
+  if (!config.tagsAndSalts) {
+    done(null, config.secret);
+    return;
+  }
+
+  const issuer = config
+    .tagsAndSalts
+    .split(",")
+    .map(i => i.split(":"))
```

Comment:
One optimisation could to do the split at launch instead at each api call.
But it could be early optimisation, the test of this version should be prioritized.

Reply:
yes that's what I was working on in the meantime but it makes testing harder because it's global state and global state is hard to mock. I'll give it another try and if I can't do it we can just save it for a rainy day.